go.mod

```diff
@@ -2,4 +2,4 @@ module github.com/optable/match-api
 
 go 1.16
 
-require google.golang.org/protobuf v1.27.1
+require google.golang.org/protobuf v1.26.0-rc.1
```

Comment:
Why are we downgrading protobuf?

Reply:
Shouldn't be. I ran `go mod tidy` which for some reason set the go.mod file like this? Not exactly sure but going to revert it back to the current version. Commit [49f4f68](https://github.com/Optable/match-api/pull/18/commits/49f4f68c5ad04e3f292702ad72491791fd131bcb) resolves this

Reply:
Actually commit [06a1875](https://github.com/Optable/match-api/pull/18/commits/06a18753d6785145b3c8b5c0bf449b01fbe1d4bc) fixes it